### PR TITLE
#188 webgpu

### PR DESCRIPTION
**Pull request by WingZer0o**
#188

### CODE REVIEW

File: webgpu/webgpu-experimental.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+This [example](https://github.com/denoland/webgpu-examples/tree/main/capture) shows how to capture an image by rendering it to a texture, copying the texture to a buffer, and retrieving it from the buffer.
+
+```typescript
+import {
+  copyToBuffer,
+  createCapture,
+  createPng,
+  Dimensions,
+} from "../utils.ts";
```
Comment (kt3k) [inline webgpu/webgpu-experimental.md:36]: this import seems causing type errors in CI check

**Review by kt3k — COMMENTED** [review]
Can you sign the CLA from the above link?

Can you also format the markdown file with `deno fmt` command?

File: webgpu/webgpu-experimental.md (hunk 1 above)
Comment (WingZer0o) [reply webgpu/webgpu-experimental.md:45]: Just leaving a note for myself.

That import can be found here.
https://github.com/denoland/webgpu-examples/blob/main/utils.ts

Comment (kt3k) [reply webgpu/webgpu-experimental.md:45]: I think it's ok to import from `denoland/webgpu-examples` as it's officially maintained by denoland org. Maybe let's import from `https://github.com/denoland/webgpu-examples/blob/6c505182170ba10cc7bfe86203477688b96648bb/utils.ts`?

Comment (WingZer0o) [reply webgpu/webgpu-experimental.md]: error: Expected a JavaScript or TypeScript module, but identified a Unknown module. Importing these types of modules is currently not supported.
  Specifier: https://github.com/denoland/webgpu-examples/blob/6c505182170ba10cc7bfe86203477688b96648bb/utils.ts
    at file:///home/runner/work/manual/manual/webgpu/webgpu-experimental.md$39-80.ts:6:8
Error: Process completed with exit code 1.

**Comment by kt3k** [comment]
Fixed the type errors.

Can you sign the CLA from the above link? @WingZer0o

**Comment by WingZer0o** [comment]
I signed a couple times, not to sure why it is saying I have not.

**Comment by kt3k** [comment]
Looks like your github account is connected to a different email you used for committing to this branch. I updated the author email of this branch

**Review by kt3k — APPROVED** [review]
LGTM

**Pull request merged**